services/storage.ts

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable functional/no-let */
+/* eslint-disable functional/immutable-data */
+import { BlobItem, BlobServiceClient } from "@azure/storage-blob";
+
+export const listBlobs = (
+  blobServiceClient: BlobServiceClient,
+  container: string
+): AsyncIterableIterator<BlobItem> => {
+  const containerClient = blobServiceClient.getContainerClient(container);
+
+  return containerClient.listBlobsFlat();
+};
+
+export const getLastBlob = async (
+  blobServiceCliet: BlobServiceClient,
+  container: string
+): Promise<BlobItem | undefined> => {
+  let res;
+
+  for await (const blob of listBlobs(blobServiceCliet, container)) {
```

Comment:
sounds good, but we could create a blob naming convention based on a timestamp, to avoid `for`. As an improvement to do in another PR  (if you are agree cc @MarcoAGreco )

Reply:
I agree, it seems a better solution 👍